earth_enterprise/src/common/serverdb/serverdbReader.cpp

```diff
@@ -396,8 +398,11 @@ const MimeType ServerdbReader::GetData(
     } else if (request == "Json") {
       std::string json_variable_name = arg_map["var"];
       // CheckJavascriptName(json_variable_name);
-      GetJson(json_variable_name, language, region, buf);
+      GetJavaScript(json_variable_name, language, region, buf);
       return MapTileUtils::kJavascriptMimeType;
+    } else if (request == "JsonOnly") {
+      GetJSON(language, region, buf);
```

Comment:
Style: To be consistent with other names, this function name would be `GetJson`.